### DISCUSSION TITLE
new script input shaping

### DISCUSSION THREAD

**Discussion by MingDestro** [Guilouz/Creality-Helper-Script-Wiki#427] [General]
Hi, when doing input shaping both X and Y type are zv instead of x=ei and y=zv. Is it ok for both type to be of the same or different?

**Comment by Guilouz** [comment]
Values depending of shaper result, you can have the same or not.

**Reply by MingDestro** [reply]
![Capture](https://github.com/Guilouz/Creality-K1-Series/assets/163025064/b1deb2df-3494-4dd0-89e4-2fe048baba25)
 so it is fine to have different values but **both shaper_type_X and shaper_type_Y to be both ZV**, is it ok too?
![Capture](https://github.com/Guilouz/Creality-K1-Series/assets/163025064/7d1e3ba6-632d-42e4-ba78-d87846ef9a6d)
